test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -486,7 +486,7 @@ def _find_module_utils(self, main):
                     pass
                 names.extend([n.name for n in child.names])
 
-                if [n for n in names if n.startswith('ansible.module_utils')]:
+                if [n for n in names if n.startswith('ansible.module_utils') or (self.collection and 'plugins.module_utils' in n)]:
```

Comment:
Just to clarify, even if a module is in a collection, it is still value to import from `ansible.module_utils`, right?

For example

```python
from ansible.module_utils.six.moves.urllib.parse import urlencode
```

I think this is still allowed by this PR (because of the `or`), so 👍

Reply:
:+1: